home/src/main/resources/rdf/i18n/de_DE/display/firsttime/search_individuals_vitro.n3

```diff
@@ -12,3 +12,7 @@
 :field_type rdfs:label "Typ"@de-DE .
 :filter_querytext rdfs:label "Text"@de-DE .
 :field_querytext rdfs:label "Standardfeld"@de-DE .
+:sort_name_raw_desc rdfs:label "Roher Titel Z-A"@de-DE .
+:sort_name_raw_asc rdfs:label "Roher Titel A-Z"@de-DE .
+:field_score rdfs:label "Relevanzfeld"@de-DE .
+:field_name_raw rdfs:label "Rohes Titelfeld"@de-DE .
```

Comment:
Where those two labels are used?

Reply:
Those labels were created for the same reason as described above. For future work with search settings in web interface.

Reply:
ok, let's keep these labels. I will organize validation of messages by native speakers